### PR TITLE
correct path for sourceContent

### PR DESCRIPTION
**Pull request by Psvensso**
Could not get the sourceContent to work, 
figured out that the source-map didnt like the path. 

Mopped up a test and changed the relative to our rel path.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -131,4 +131,53 @@ describe('gulp-concat-sourcemap', function() {
         stream.write(file2);
         stream.end();
     });
+	
+	it('should generate source content inline', function(){
```
Comment (mikach) [inline test/test.js]: Seems like you forgot to pass `done` to function?

**Comment by mikach** [comment]
Please, take a look at my comment on the diff.

**Comment by Psvensso** [comment]
There we go.... :)

**Comment by mikach** [comment]
Thanks!

**Pull request merged**